apps/logging/serializers.py

```diff
@@ -135,6 +144,17 @@ def application(self):
     def path(self):
         return self.req.headers.get('BlueButton-OriginalUrl')
 
+    def to_dict(self):
+        result = {
+            "uuid": self.uuid(),
+            "fhir_id": self.fhir_id(),
+            "user": self.user(),
+            "start_time": self.start_time(),
+            "application": self.application(),
+            "path": self.path(),
+        }
+        return result
```

Comment:
@JFU-GIT Wondering if now would be a good time to update the related logging in JSON format for Splunk ingestion?
For example, related to this serializer dictionary the following could be changed in apps.logging.signals.py:

```python
def fetching_data(sender, request=None, **kwargs):
    fhir_logger.info(json.dumps(FHIRRequest(request)))
```

Reply:
good catch, will apply json dump.